packages/next/src/lib/scheduler.ts

```diff
@@ -32,3 +32,7 @@ export const scheduleImmediate = <T = void>(cb: ScheduledFn<T>): void => {
     setImmediate(cb)
   }
 }
+
+export function atLeastOneTask() {
```

Comment:
It's named so that `await atLeastOneTask()` tells you semantically what you are trying to do. I want the code after this await to guaranteed be after the current microtask queue is flushed. This is also useful if you want to Promise.race something against the completion of the current Task.

added comments